src/proxies/panel_manager.js

```diff
@@ -89,6 +89,14 @@ PanelManager.openCategory = async function (options) {
   })
 }
 
+PanelManager.refreshCategoryPois = async function () {
```

Comment:
Instead of looping over all the panels, it will be better to fire an event when the move is finished (in `scene.js`) and listen that event on `CategoryPanel`. With that we can plug anything on that event later if we need and this avoid to loop over every panels on every moves.

Reply:
Done